modules/Schedulers/_AddJobsHere.php

```diff
@@ -75,7 +77,7 @@
     12 => 'sendEmailReminders',
     14 => 'cleanJobQueue',
     15 => 'removeDocumentsFromFS',
-    16 => 'trimSugarFeeds',
```

Comment:
This concerns me.. will this affect a database change i.e. will schedules fail to run now because we have changed the name?